jujugui/static/gui/src/app/components/profile/model-list/test-model-list.js

```diff
@@ -0,0 +1,231 @@
+    "name": "mymodel2",
+    "series": "xenial",
+    "provider": "gce",
+    "uuid": "f507b811-d706-427e-8f01-d485fgc90a67",
+    "agentVersion": "",
+    "sla": "",
+    "slaOwner": "",
+    "status": "available",
+    "statusInfo": "",
+    "controllerUUID": "a030379a-940f-4760-8fce-3062b41a04e9",
+    "owner": "tester@external",
+    "credential": "google_tester@external_admin",
+    "credentialName": "admin",
+    "region": "us-central1",
+    "cloud": "google",
+    "numMachines": 0,
+    "users": [{
+      "name": "tester@external",
+      "displayName": "tester",
+      "domain": "Ubuntu SSO",
+      "lastConnection": "2017-07-06T16:26:47.000Z",
+      "access": "admin"
+    }],
+    "life": "alive",
+    "isAlive": true,
+    "isController": false,
+    "lastConnection": "2017-07-06T16:26:47.000Z"
+  }, {
+    "id": "06b11eb5-abd6-48f3-8910-b54cf5945e60",
+    "name": "test-db",
+    "series": "xenial",
+    "provider": "gce",
+    "uuid": "06b11eb5-abd6-48f3-8910-b54cf5945e60",
+    "agentVersion": "",
+    "sla": "",
+    "slaOwner": "",
+    "status": "available",
+    "statusInfo": "",
+    "controllerUUID": "a030379a-940f-4760-8fce-3062b41a04e9",
+    "owner": "tester2@external",
+    "credential": "google_tester2@external_google-dbtest",
+    "credentialName": "google-dbtest",
+    "region": "us-east1",
+    "cloud": "google",
+    "numMachines": 2,
+    "users": [{
+      "name": "tester@external",
+      "displayName": "tester",
+      "domain": "Ubuntu SSO",
+      "lastConnection": null,
+      "access": "read"
+    }],
+    "life": "alive",
+    "isAlive": true,
+    "isController": false,
+    "lastConnection": null
+  }, {
+    "id": "509f6e4c-4da4-49c8-8f19-537c33b4d3a0",
+    "name": "website",
+    "series": "xenial",
+    "provider": "gce",
+    "uuid": "509f6e4c-4da4-49c8-8f19-537c33b4d3a0",
+    "agentVersion": "",
+    "sla": "",
+    "slaOwner": "",
+    "status": "available",
+    "statusInfo": "",
+    "controllerUUID": "a030379a-940f-4760-8fce-3062b41a04e9",
+    "owner": "tester3@external",
+    "credential": "google_tester3@external_gce",
+    "credentialName": "gce",
+    "region": "us-east1",
+    "cloud": "google",
+    "numMachines": 25,
+    "users": [{
+      "name": "tester4@external",
+      "displayName": "tester4",
+      "domain": "Ubuntu SSO",
+      "lastConnection": "2017-06-09T16:24:28.000Z",
+      "access": "admin"
+    }, {
+      "name": "tester@external",
+      "displayName": "tester",
+      "domain": "Ubuntu SSO",
+      "lastConnection": "2017-07-05T01:42:05.000Z",
+      "access": "admin"
+    }],
+    "life": "alive",
+    "isAlive": true,
+    "isController": false,
+    "lastConnection": "2017-07-05T01:42:05.000Z"
+  }]`;
+
+  beforeAll(function(done) {
+    // By loading this file it adds the component to the juju components.
+    YUI().use('profile-model-list', function() {
+      done();
+    });
+  });
+
+  function renderComponent(options={}) {
```

Comment:
Maybe add a modelResponse parameter, defaulting to rawModelData?
This way we can easily test the cases in which we have no owned models, no shared models, and no models at all.